src/main/java/qna/domain/BaseTime.java

```diff
@@ -0,0 +1,31 @@
+package qna.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public abstract class BaseTime {
+
+    @CreatedDate
+    @Column(nullable = false, updatable = false)
+    private LocalDateTime createdAt;
+
+    @LastModifiedDate
+    @Column(nullable = false)
+    private LocalDateTime updateAt;
```

Comment:
사소한 부분이긴 하지만 단순 실수하신 것 같아요 😅

```suggestion
    @Column
    @LastModifiedDate
    private LocalDateTime updatedAt;
```

Reply:
다음번에는 좀더 신경쓰겠습니다. 😢